app/helpers/puppetclasses_and_environments_helper.rb

```diff
@@ -28,12 +28,21 @@ def pretty_print(classes)
       if (mod = klass.gsub(/::.*/, ""))
         hash[mod] ||= []
         hash[mod] << klass
-      else
-        next
       end
     end
     hash.keys.sort.map do |key|
-      link_to key,{}, {:remote => true, :rel => "popover", :data => {"content" => hash[key].sort.join('<br>').html_safe, "original-title" => key}}
+      @classes_str = "classes"
+      @num = hash[key].size
+      @obj_name = @num > 1 ? @classes_str : @classes_str.singularize
```

Comment:
This only works for English, instead get rid of this and for the original-title below do:

    n_("%{name} has ... %{num} class", "%{name} has ... %{num} classes", @num)  % {:name => key, :num => @num }

This translates both plural strings and works for languages with more complex rules too.